release/models/bgp/openconfig-bgp-types.yang

```diff
@@ -595,6 +601,25 @@ module openconfig-bgp-types {
                 '429[0-3][0-9]{6}|42[0-8][0-9]{7}|4[0-1][0-9]{8}|[1-3][0-9]{9}|'  +
                 '[1-9][0-9]{1,8}|[0-9])$';
       }
+      type string {
+        // Extended Link Bandwidth Community
+        // bandwidth:<2 byte asn>:<bandwidth value in bytes/sec>
+        pattern 'bandwidth:'                                                      +
```

Comment:
nit, we think this might be better served as `link-bandwidth` vs. `bandwidth`.

Reply:
Done